packages/blockly-field-date/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@blockly/field-date",
+    "version": "3.20200304.0",
+    "description": "A Blockly date picker field that uses the Google Closure date picker.",
+    "scripts": {
+        "build": "gulp build"
+    },
+    "main": "build/date_compressed.js",
+    "author": "Blockly Team",
+    "keywords": [
+        "blockly",
+        "field",
+        "date",
+        "datepicker"
+    ],
+    "bugs": {
+        "url": "https://github.com/google/blockly-samples/issues"
+    },
+    "license": "Apache-2.0",
+    "peerDependencies": {
+        "blockly": ">=3.20191014.4"
```

Comment:
should there be an upper bound here?

Reply:
No need. This field should be compatible with all future releases.